### PR TITLE
Rc1

### PR DESCRIPTION
**Pull request by linxie47**

### CODE REVIEW

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ jobs:
           TRIVY_JAVA_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-java-db
         with:
           scan-type: 'image'
-          image-ref: 'ffmpeg_ivsr_sdk_ov2022.3'
+          image-ref: 'ffmpeg_ivsr_sdk_ubuntu22_ov2024.5s'
```
Comment (linxie47) [inline .github/workflows/trivy.yml:58]: @wangjingz docker image name is changed, so i made changes for trivy.

File: .github/workflows/ubuntu-build-docker.yml [hunk 2]
```diff
@@ -28,4 +28,4 @@ jobs:
           sudo systemctl daemon-reload
           sudo systemctl restart docker
           cd ./ivsr_ffmpeg_plugin
-          ./build_docker.sh --enable_ov_patch true --ov_version 2022.3
+          ./build_docker.sh --ov_version 2024.5s --os_version rockylinux9
```
Comment (linxie47) [inline .github/workflows/ubuntu-build-docker.yml:31]: choose a simpler image that can be built faster.

**Review by xiaoxial — APPROVED** [review]
LGTM

File: .github/workflows/ubuntu-build-docker.yml (hunk 2 above)
Comment (djie1) [reply .github/workflows/ubuntu-build-docker.yml:31]: How about renaming the name of this file from "ubuntu-build-*" to "rocky-build-*"?

File: README.md [hunk 3]
```diff
@@ -69,36 +75,39 @@ Currently, iVSR offers two AI media processing functionalities: Video Super Reso
...
+  The input shape of this model is `[1, (channels * frames)9, H, W]`, and the output shape is `[1, (channels)3, 2xH, 2xW]`. For each inference, the input data is the `(n-1)th`, `(n)th`, and `(n+1)th` frames combined. The output data is the `(N)th` frame. For the first frame, the input data is `1st`, `1st`, `2nd` frames combined. For the last frame, the input data is the `(n-1)th`, `(n)th`, `(n)th` frames combined.<br>
 
 ### 1.4.2. Smart Video Processing (SVP)
-`SVP` is an AI-based video prefilter that enhances the perceptual rate-distortion in video encoding. With `SVP`, the encoded video streams maintain the same visual quality while reducing bandwidth, as measured by common video quality metrics (such as VMAF and (MS-)SSIM) and human perception.
+`SVP` is an AI-based video prefilter that enhances the perceptual rate-distortion in video encoding. With `SVP`, the encoded video streams maintain the same visual quality while reducing bandwidth, as measured by common video quality metrics (such as VMAF and (MS-)SSIM) and human perception.<br><br>
+The input and output shape of this model is `[1, (channels)3, H, W]`.<br>
```
Comment (djie1) [inline README.md:95]: Just found we missed the description of SVP-Basic and SVP-SE model variances. And we also have to update the description of the shapes supported.

File: build_ivsr.sh [hunk 4]
```diff
@@ -92,8 +92,8 @@ apt-get update && DEBIAN_FRONTEND=noninteractive && apt-get install -y --no-inst
         python3-dev libpython3-dev python3-pip
 
 apt-get clean
-pip --no-cache-dir install --upgrade pip setuptools
-pip install numpy
+pip --no-cache-dir install --upgrade pip==23.0 setuptools==65.5.0
+pip install numpy==1.23.5
```
Comment (djie1) [inline build_ivsr.sh:96]: looks like this change is not in inner-source repo?

File: ivsr_sdk/privates/model_guard.bin/libirguard.a-2024.4.0
Comment (djie1) [inline ivsr_sdk/privates/model_guard.bin/libirguard.a-2024.4.0:1]: Do we still need this file?

Comment (linxie47) [reply ivsr_sdk/privates/model_guard.bin/libirguard.a-2024.4.0:1]: can be removed

File: build_ivsr.sh (hunk 4 above)
Comment (linxie47) [reply build_ivsr.sh:96]: yes will cherry-pick back to inner

**Review by djie1 — APPROVED** [review]
LGTM

**Review by xiaoxial — APPROVED** [review]

**Pull request merged**